src/components/Organisms/Sidebar.jsx

```diff
@@ -0,0 +1,73 @@
+import { FaBriefcase } from "react-icons/fa";
+import { Button } from "../Molecules/Button";
+import { IoIosSchool } from "react-icons/io";
+import { IoMdSettings } from "react-icons/io";
+import { CiLogout } from "react-icons/ci";
+import { AiFillHome } from "react-icons/ai";
+
+export const Sidebar = ({ log }) => {
+  return (
+    <div className="min-h-16 flex flex-row items-center justify-center gap-x-7 bg-primary-500 py-3 md:h-screen md:w-64 md:flex-col md:items-center md:justify-center md:gap-x-0 md:space-y-5 md:px-8">
+      <Button
+        icon={<AiFillHome />}

```

Comment:
Se repiten las lineas:
color="bg-primary-500"
position={"justify-around"}
iconColor={"white"}
iconSize={"1.25em"}
sizeSM_w={"w-10"}
sizeSM_h={"h-10"}
sizeMD_w={"md:w-36"}
sizeMD_h={"md:h-14"}

ver posibilidad de refactorizar esta parte.

Reply:
👎

Reply:
> 

Sorry cant understando your palabrations

Reply:
Oh, sorry for that my friend